kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -1,24 +1,38 @@
 // $(Exercises).trigger("clearExistingProblem");
 
-var KhanUtil = window.KhanUtil || {
-    debugLog: function() {}
+var icu = {
+    getDecimalFormatSymbols: function() {
+        return {
+            decimal_separator: ".",
+            grouping_separator: ",",
+            minus: "-"
+        };
+    },
+    getLanguage: function() {
+        return "en";
+    }
 };
 
+
+var KhanUtil = window.KhanUtil || {};
+
 var Khan = window.Khan || {
-    Util: KhanUtil,
     error: function() {},
     query: {debug: ""},
     imageBase: STATIC_URL + "perseus/ke/images/",
+    urlBase: STATIC_URL + "perseus/ke/",
```

Comment:
Due to changes in Khan Exercises (introduced by us), we can now define whatever we want on the Khan object prior to loading and have those changes remain extant.